content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -260,6 +255,19 @@ The above update rules apply to regular pod updates, but other pod fields can be
 - **Binding:** The `binding` subresource allows setting the pod's `spec.nodeName` via a `Binding` request.
   This is typically only used by the {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}}.
 
+### Pod generation
+
+- The `generation` field is unique. It will be automatically set by the
+  system such that new pods have a `metadata.generation` of 1, and every update to
+  mutable fields in the pod's spec will increment the `metadata.generation` by 1.
+
+  {{< feature-state for_k8s_version="v1.34" state="beta" >}}
+
+- `observedGeneration` is an optional field that is captured in the `status` section of the Pod
```

Comment:
nit: remove the word "optional". Optional usually implies that it is optional for the user, but in this case the field is set by the system (and is always set when the feature gate is on)

```suggestion
- `observedGeneration` is a field that is captured in the `status` section of the Pod
```